Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "nickel_postgres"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
These version changes are probably a breaking change, so let's take this up to 0.3.0

Reply:
Yeah it should be 0.3.0